.github/workflows/notify.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger Publish
+
+on:
+  push:
+    paths:
+      - 'doc/docs'
+    branches:
+      - '4.0'
+
+jobs:
+  trigger_publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger Developer Event
+        uses: peter-evans/repository-dispatch@master
+        with:
+          token: ${{ secrets.BUILD_ACCESS_TOKEN }}
```

Comment:
@jexp @adam-cowley Could you please make sure that the token `BUILD_ACCESS_TOKEN` is available on this repository? Thanks!

Reply:
@adam-cowley ping ☝️